src/opcode.rs

```diff
@@ -87,6 +87,28 @@ where
     }
 }
 
+/// Returns true if the opcode modifies memory.
+/// <https://bluealloy.github.io/revm/crates/interpreter/memory.html#opcodes>
+/// <https://github.com/crytic/evm-opcodes>
+#[inline]
+pub const fn modifies_memory(opcode: OpCode) -> bool {
```

Comment:
we can keep this here and then remove with the next revm release if it has this function natively